common/config/babel/.babelrc.js

```diff
@@ -58,8 +58,19 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // feature for blurred background and replace background effects
         'video-background-effects',
         // feature for vertical gallery layouts in VideoGallery
-        'vertical-gallery'
+        'vertical-gallery',
+        // a demo feature flag for those beta feature not ready for beta release
+        'in-progress-beta-feature-demo'
```

Comment:
Do we need to communicate how and when to use this flag?

Reply:
I think its worthwhile to add some documentation on how to use this just so there isn't any miscommunication about adding the entry to the wrong list or how to debug something wierd going on. Getting a first version of thoughts down might be good to help us understand if there are any gaps in documentation. @PorterNan

Reply:
Definitely, we are doing this as an experiment for next release, and will have a more official version after that, once we polish this well and make this ready for general usage, we will have have a meeting talk about this and collect more feedbacks